src/apps/demo-app/e2e/BUILD.bazel

```diff
@@ -0,0 +1,45 @@
+load("@build_bazel_rules_typescript//:defs.bzl", "ts_library")
+#load("@angular//:index.bzl", "protractor_web_test_suite")
+
+ts_library(
+    name = "e2e_lib",
+    testonly = 1,
+    srcs = glob(["src/apps/demo-app/src/**/*.ts"]),
+    tsconfig = ":tsconfig.e2e.json",
+    deps = [
+      "@layoutdeps//@types/jasminewd2",
+      "@layoutdeps//jasmine",
+      "@layoutdeps//protractor",
+    ]
+)
+
+#protractor_web_test_suite(
+#    name = "prodserver_test",
+#    data = [
+#      "@angular//packages/bazel/src/protractor/utils",
+#      "@layoutdeps//protractor",
+#    ],
+#    on_prepare = ":protractor.on-prepare.js",
+#    server = "//src:prodserver",
+#    deps = [":e2e_lib"],
+#)
+
+#protractor_web_test_suite(
+#    name = "devserver_test",
+#    data = [
+#      "@angular//packages/bazel/src/protractor/utils",
+#      "@layoutdeps//protractor",
+#    ],
+#    on_prepare = ":protractor.on-prepare.js",
+#    server = "//src/apps/demo-app/src:devserver",
+#    deps = [":e2e_lib"],
+#    tags = ["manual"],
+#)
+
+# Default target in this package is to run the e2e tests on the devserver.
+# This is a faster round-trip but doesn't exercise production optimizations like
+# code-splitting and lazy loading.
+#alias(
+#    name = "e2e",
+#    actual = "devserver_test",
+#)
```

Comment:
commented stuff should be removed. Even though if it may be re-added in future PRs